### PR TITLE
WIP: [docs] VSphere golden image for airgapped network

### PR DESCRIPTION
**Pull request by jrvaldes**
Include instructions to pre-pull the Pause Image into the golden
image during the creation process, to support disconnected network
environments.

Follow-up for #579

### CODE REVIEW

File: docs/vsphere-golden-image.md [hunk 1]
```diff
@@ -129,6 +129,17 @@ Install the `docker` container runtime following the [Microsoft's documentation]
 Alternatively, you can run the provided PowerShell script [install-docker.ps1](vsphere_ci/scripts/install-docker.ps1) 
 to programmatically install `docker` in the Windows VM.
 
+### Set up for disconnected network environment
```
Comment (saifshaikh48) [inline docs/vsphere-golden-image.md:132]: Could someone using another cloud provider also be interested in this step? Just wondering if this should be in a more generic place (like the top-level README)

Comment (jrvaldes) [reply docs/vsphere-golden-image.md:132]: Good point, given the creation of a golden image is only a pre-req for vSphere; here is definitely a good fit for this.
As per OpenShift [docs](https://docs.openshift.com/container-platform/4.8/installing/installing-preparing.html#supported-installation-methods-for-different-platforms), _restricted network_ is also supported in AWS with IPI, not the case for Azure.

**Comment by aravindhp** [comment]
/hold

until the actual work can be prioritized and tested

**Pull request closed without merging**